pontoon/administration/views.py

```diff
@@ -182,12 +182,8 @@ def manage_project(request, slug=None, template="admin_project.html"):
                     .delete()
                 )
 
-                project_locales = [
-                    ProjectLocale(project=project, locale=locale) for locale in locales
-                ]
-                ProjectLocale.objects.bulk_create(
-                    project_locales, ignore_conflicts=True
-                )
+                for locale in locales:
+                    ProjectLocale.objects.get_or_create(project=project, locale=locale)
```

Comment:
Could you add a comment here why don't we use `bulk_create` here, to avoid making the same mistake in the future?

Reply:
```suggestion
                    # The implicit pre_save and post_save signals sent here are required
                    # to maintain django-guardian permissions.
                    ProjectLocale.objects.get_or_create(project=project, locale=locale)
```